class.jetpack-data.php

```diff
@@ -39,4 +39,87 @@ public static function get_access_token( $user_id = false ) {
 			'external_user_id' => (int) $user_id,
 		);
 	}
+
+	/**
+	 * This function mirrors Jetpack_Data::is_usable_domain() in the WPCOM codebase.
+	 *
+	 * @param $domain
+	 * @param array $extra
+	 *
+	 * @return bool|WP_Error
+	 */
+	public static function is_usable_domain( $domain, $extra = array() ) {
+
+		// If it's empty, just fail out.
+		if ( ! $domain ) {
+			return new WP_Error( 'fail_domain_empty', sprintf( 'Domain `%1$s` just failed is_usable_domain check as it is empty.', $domain ) );
```

Comment:
Shouldn't we make all these strings translateable by adding the `jetpack` textdomain?